web/src/components/overview/OverviewPage.tsx

```diff
@@ -90,19 +81,13 @@ const IssuesList = ({ issues }: { issues: IssuesListType }) => {
   );
 };
 
-const ResultSection = () => {
-  const issues = useAllIssues();
-
-  const resultSectionProps = issues.isEmpty
-    ? {}
-    : {
-        title: _("Installation"),
-        description: _("Before installing, please check the following problems."),
-      };
-
+const IssuesSection = ({ issues }: { issues: IssuesListType }) => {
   return (
-    <Page.Section {...resultSectionProps}>
-      {issues.isEmpty ? <ReadyForInstallation /> : <IssuesList issues={issues} />}
+    <Page.Section
+      title={_("Installation blocking issues")}
+      description={_("Before installing, please check the following problems.")}
```

Comment:
:sos: Another place where I need help with texts. But please, do not invest too much time because if we reach an agreement such a section will dissapear from the Overview page soon.